firebase-testlab-instr-lib/testlab-instr-lib/build.gradle

```diff
@@ -21,31 +20,17 @@ android {
 }
 
 dependencies {
-    implementation 'junit:junit:4.12'
-    implementation 'androidx.test.ext:junit:1.1.2'
     implementation 'androidx.test:runner:1.3.0'
 }
 
 group = 'com.google.firebase'
 version = '0.2'
 
-uploadArchives {
```

Comment:
So we want to upload step?

Reply:
Since we need to publish through google which requires internal permissions we use the artifacts created in publishMavenPublicationToMavenLocal instead